### ISSUE TITLE
Correction in post-migration tasks

### ISSUE THREAD

**Issue by ohannanen** [MicrosoftDocs/OfficeDocs-Exchange#99]
Current note in step 7 states "In Exchange 2007, Exchange 2010, and Exchange 2013 you should also set Set-ClientAccessServer AutodiscoverInternalConnectionURI to Null."

In Exchange 2010 and 2013, the referenced parameter is called AutoDiscoverServiceInternalUri, not AutodiscoverInternalConnectionURI. I have not confirmed in Exchange 2007. 

I think it would be helpful to list the full command to avoid confusion:
Set-ClientAccessServer -identity &lt;CasServerName&gt; -AutoDiscoverServiceInternalUri $Null

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 721f096d-23d4-0fed-4f5e-0e0964798e8f
* Version Independent ID: 3af25c86-f5e9-1b9c-ffb6-d0d1afa7210e
* Content: [Migrate email using the Exchange cutover method](https://docs.microsoft.com/en-us/Exchange/mailbox-migration/cutover-migration-to-office-365?redirectSourcePath=%252fen-gb%252farticle%252fcutover-migration-to-office-365-9496e93c-1e59-41a8-9bb3-6e8df0cd81b4)
* Content Source: [Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md)
* Service: **exchange-online**
* Product: **exchange**
* GitHub Login: @msdmaguire
* Microsoft Alias: **dmaguire**

**Comment by AndreaBarr** [comment]
@ohannanen Thank you for this feedback.

I agree it would be helpful to have the full command in there. I am getting this to the SharePoint team so we can get the addressed properly. 

I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
